webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/list/ActivityStreamList.vue

```diff
@@ -228,6 +228,17 @@ export default {
     loadActivities(newActivitiesCount) {
       this.limit += newActivitiesCount;
       this.loadActivityIds().catch(() => window.location.reload());
+      this.$nextTick().then(() => {
+        const $streamPageContainer = $('#ActivityStream')[0];
```

Comment:
```suggestion
        const streamPageContainerElement = document.getElementById('ActivityStream');
```
it's better to not use jQuery anymore, we want to deprecate it.

Reply:
Updated!